pylint_odoo/checkers/no_modules.py

```diff
@@ -342,6 +355,10 @@ def visit_assign(self, node):
     def camelize(self, string):
         return re.sub(r"(?:^|_)(.)", lambda m: m.group(1).upper(), string)
 
+    def formatversion(self, string):
+        regx = r"(\d*[6-9])(\.\d*[0-9])(\.\d*[0-9])(\.\d*[0-9])(\.\d*[0-9])"
```

Comment:
Move this regex to configuration file of pylint adding to `options` variable.
With the default previous regex